README.md

```diff
@@ -28,3326 +28,235 @@
+###### [Caveats when using collections polyfill](docs/features/ecmascript/collections.md#caveats-when-using-collections-polyfill)
+
+##### [ECMAScript: Typed Arrays](docs/features/ecmascript/typed-array.md)
+###### [Caveats when using typed arrays polyfills](docs/features/ecmascript/typed-array.md#caveats-when-using-typed-arrays-polyfills)
+##### [ECMAScript: Reflect](docs/features/ecmascript/reflect.md)
+##### [ECMAScript: JSON](docs/features/ecmascript/json.md)
+##### [ECMAScript: globalThis](docs/features/ecmascript/global-this.md)
+
+#### [ECMAScript proposals](docs/features/proposals/README.md)
+##### [Finished proposals](docs/features/proposals/README.md#finished)
+###### [`globalThis`](docs/features/proposals/global-this.md)
+###### [Relative indexing method](docs/features/proposals/relative-indexing-method.md)
+###### [`Array.prototype.includes`](docs/features/proposals/array-includes.md)
+###### [`Array.prototype.flat` / `Array.prototype.flatMap`](docs/features/proposals/array-flat-map.md)
+###### [Array find from last](docs/features/proposals/array-find-from-last.md)
+###### [`Object.values` / `Object.entries`](docs/features/proposals/object-values-entries.md)
+###### [`Object.fromEntries`](docs/features/proposals/object-from-entries.md)
+###### [`Object.getOwnPropertyDescriptors`](docs/features/proposals/object-getownpropertydescriptors.md)
+###### [Accessible `Object.prototype.hasOwnProperty`](docs/features/proposals/accessible-object-hasownproperty.md)
+###### [`String` padding](docs/features/proposals/string-padding.md)
+###### [`String#matchAll`](docs/features/proposals/string-match-all.md)
+###### [`String#replaceAll`](docs/features/proposals/string-replace-all.md)
+###### [`String.prototype.trimStart` / `String.prototype.trimEnd`](docs/features/proposals/string-left-right-trim.md)
+###### [`RegExp` `s` (`dotAll`) flag](docs/features/proposals/regexp-dotall-flag.md)
+###### [`RegExp` named capture groups](docs/features/proposals/regexp-named-groups.md)
+###### [`Promise.allSettled`](docs/features/proposals/promise-all-settled.md)
+###### [`Promise.any`](docs/features/proposals/promise-any.md)
+###### [`Promise.prototype.finally`](docs/features/proposals/promise-finally.md)
+###### [`Symbol.asyncIterator` for asynchronous iteration](docs/features/proposals/async-iteration.md)
+###### [`Symbol.prototype.description`](docs/features/proposals/symbol-description.md)
+###### [Well-formed `JSON.stringify`](docs/features/proposals/well-formed-stringify.md)
+
+##### [Stage 3 proposals](docs/features/proposals/README.md#stage-3)
+###### [`Array` grouping](docs/features/proposals/array-grouping.md)
+###### [Change `Array` by copy](docs/features/proposals/change-array-by-copy.md)
+
+##### [Stage 2 proposals](docs/features/proposals/README.md#stage-2)
+###### [Iterator helpers](docs/features/proposals/iterator-helpers.md)
+####### [Caveats](docs/features/proposals/iterator-helpers.md#caveats)
```

Comment:
![image](https://user-images.githubusercontent.com/2213682/182033406-3d620029-003b-4c99-878c-33de9b9c696b.png)

Too low header.

![image](https://user-images.githubusercontent.com/2213682/182033450-5b1947cf-b626-40a5-9caf-881b229f6383.png)

And is not readable below - it's better to use something else, for example, lines, for split sections.